app/views/layouts/application.html.erb

```diff
@@ -37,33 +37,28 @@
       <li><strong><i class="fas fa-car-side"></i><%= link_to "My Rides", "#"%></strong></li>
 
         <% if user_signed_in? %>
-        <li><strong><i class="fas fa-sign-out-alt"></i>
+          <li><strong><i class="fas fa-user"></i>
```

Comment:
I'm sure this has something to do with Devise, but can you explain the "fas fa-xx" stuff to me?

Reply:
user_signed_in? is a devise method. <i class="fas fa-sign-out-alt"></i> is FontAwesome code that we used for icons. 
You can name your review pages however you like depending on how you are would like to style the page or how you are writing your feature tests. I'm also fine with waiting and doing styling together when we can work together in person.